clang/examples/DPCT/Runtime/__activemask.cu

```diff
@@ -0,0 +1,5 @@
+__device__ void test(unsigned int r) {
+  // Start
+  r = __activemask();
```

Comment:
The mapping for this API is not exactly correct, we need put DPCT1086 in mapping. 
Please sync with Jiajun on the final update.

Reply:
Synced with @tangjj11 the warning will be remain.